src/pages/workspaces/workspace/Data.js

```diff
@@ -114,86 +553,39 @@ const WorkspaceData = _.flow(
     try {
       const entityMetadata = await Workspaces.workspace(namespace, name).entityMetadata()
       this.setState({
-        selectedDataType: this.isDataModel() && !entityMetadata[selectedDataType] ? undefined : selectedDataType,
+        selectedDataType: this.selectionType() === 'entities' && !entityMetadata[selectedDataType] ? undefined : selectedDataType,
         entityMetadata
       })
     } catch (error) {
       reportError('Error loading workspace entity data', error)
     }
   }
 
-  async loadData() {
-    const { namespace, name, ajax: { Workspaces } } = this.props
-    const { itemsPerPage, pageNumber, sort, selectedDataType } = this.state
-    const isDataModel = this.isDataModel()
-
-    const getWorkspaceAttributes = async () => (await Workspaces.workspace(namespace, name).details()).workspace.attributes
-
-    if (!selectedDataType) {
-      this.setState({ workspaceAttributes: await getWorkspaceAttributes() })
-    } else {
-      try {
-        this.setState({ loading: true })
-
-        const [workspaceAttributes, entities] = await Promise.all([
-          getWorkspaceAttributes(),
-          isDataModel ?
-            Workspaces.workspace(namespace, name)
-              .paginatedEntitiesOfType(selectedDataType, {
-                page: pageNumber, pageSize: itemsPerPage, sortField: sort.field, sortDirection: sort.direction
-              }) :
-            undefined
-        ])
-
-        this.setState(_.merge(
-          isDataModel && { entities: entities.results, totalRowCount: entities.resultMetadata.unfilteredCount, selectedEntities: [] },
-          { workspaceAttributes }
-        ))
-      } catch (error) {
-        reportError('Error loading workspace data', error)
-      } finally {
-        this.setState({ loading: false })
-      }
-    }
-  }
-
-  getReferenceData() {
-    const { workspaceAttributes } = this.state
-
-    return _.flow(
-      _.toPairs,
-      _.filter(([key]) => key.startsWith('referenceData-')),
-      _.map(([k, value]) => {
-        const [, datum, key] = /referenceData-([^-]+)-(.+)/.exec(k)
-        return { datum, key, value }
-      }),
-      _.groupBy('datum')
-    )(workspaceAttributes)
-  }
-
   async componentDidMount() {
     this.loadMetadata()
-    this.loadData()
-
-    this.setState({ orchestrationRoot: await Config.getOrchestrationUrlRoot() })
+    this.setState({ firstRender: false })
   }
 
   refresh() {
-    this.loadMetadata()
-    this.loadData()
+    this.setState(({ refreshKey }) => ({ refreshKey: refreshKey + 1 }))
```

Comment:
This is a way of communicating with the child component. We could use a ref, but we removed ref forwarding, so we can't reach through our HOC wrappers.